corehq/apps/app_manager/templates/app_manager/profile.xml

```diff
@@ -28,7 +28,9 @@
     {% if support_email %}
     <property key="support-email-address" value="{{ support_email }}" force="true"/>
     {% endif %}
-    <property key="cc_persistent_menu" value="{{ app.persistent_menu }}" force="true"/>
+
+    <property key="cc_persistent_menu" value="{{ app.persistent_menu|yesno:"yes,no" }}" force="true"/>
```

Comment:
@shubham1g5 I changed this one to yes/no too so they at least use the same kind of values.